apps/api/src/scraper/scrapeURL/engines/fire-engine/scrape.ts

```diff
@@ -0,0 +1,96 @@
+import { Logger } from "winston";
+import * as Sentry from "@sentry/node";
+import { z } from "zod";
+
+import { Action } from "../../../../lib/entities";
+import { robustFetch } from "../../lib/fetch";
+
+export type FireEngineScrapeRequestCommon = {
+    url: string;
```

Comment:
tlsclient also misses disableJsDom, cdp misses screenshot probably more didnt fully check

Reply:
CDP most definitely does not miss screenshot. There is a test case for it. It's transformed into an action, since you dropped the `screenshot` parameter support.

Reply:
disableJsDom is used by atleast one customer + was used by sitemap fetch at some point. We should add it and document it

Reply:
Adding...

Reply:
Added

Reply:
forgot mobileProxy, looking.

Reply:
Wait. We don't use `mobileProxy` anywhere. Not exposed to `v0`, and I got rid of the URL-specific override that used it.

Reply:
is it not exposed on v1 either?

Reply:
Nope. Do a search for `mobileProxy` on the main branch. Unused.

Reply:
@mogery it was for this site, custom case:
```
"eonhealth.com":{
    defaultScraper: "fire-engine",
    params:{
      fireEngineOptions:{
        mobileProxy: true,
        method: "get",
        engine: "request",
      },
    },
  },
  ```
  
  maybe it doesnt make sense anymore?

Reply:
It does not in my testing. Do try it out please